### ISSUE TITLE
Bug in the 'Main Menu' 

### ISSUE THREAD

**Issue by Mishahhi** [TheGameCreators/GameGuruRepo#4710]
The error lies in the fact that when changing another button name after building the build, the buttons changed by the text are not pressed. 
Maybe it's related to the code. The question is - how can this be fixed?

p.s Separately, I ask you to help me with another bug. How to fix FOV? After assembly, when you click graphic settings, the screen increases. And so with each click

**Comment by Mishahhi** [comment]
@davetgc

**Comment by Mishahhi** [comment]
@LeeBamberTGC  
Dear Lee! Please pay attention to the bug. She is quite critical of a beginner who is just starting the path of a developer. Unfortunately, it will sound rude, but the bug is quite unpleasant. It completely breaks the game menu. 
And also "FOV" is not functioning properly in the menu.

The question is, how can this be fixed? I tried for a month looking for the problem, but I never found the root of the evil

**Comment by davetgc** [comment]
I've pushed this up to high. @LeeBamberTGC - This bug isn't just affecting the menus, but if you change any of the text on any button, those buttons stop working. For example, if you change the LEAVE button on the Inventory HUD to EXIT, you can no longer exit the inventory screen. This applies to all RPG screens.

**Comment by LeeBamberTGC** [comment]
@davetgc Yes the backend logic reads the name of the button, so when the name is changed the logic does not know what HUD or EXIT means. Sounds like we need a system to TAG buttons with their logic meaning, allowing the visible button name to be renamed to anything.

**Comment by Mishahhi** [comment]
@LeeBamberTGC @davetgc  I'm sorry, but by what date can we expect these changes?
